src/test/java/org/encorapedia/EncorapediaReportPricesResultsTest.java

```diff
@@ -0,0 +1,84 @@
+package org.encorapedia;
+
+import com.sun.istack.internal.NotNull;
+import org.browser.BrowserFactory;
+import org.encorapedia.pages.EncorapediaHomePage;
+import org.encorapedia.pages.EncorapediaReportsPage;
+import org.encorapedia.pages.EncorapediaResultsPage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+import java.util.List;
+
+public class EncorapediaReportPricesResultsTest {
+    private WebDriver driver;
+
+    public void chrome() throws Exception {
+        driver= BrowserFactory.getBrowser("chrome");
+        driver.manage().window().maximize();
+        driver.get("http://localhost:8080/choice-selenium-testing/index.html");
+    }
+
+    @Before
+    public void setup() throws Exception {
+        chrome();
+    }
+
+    public void navigateToReports() {
+        EncorapediaHomePage objEncorapediaHomePage = new EncorapediaHomePage(driver);
+        objEncorapediaHomePage.clickOnReports();
+    }
+
+    @Test
+        public void validatingReportPricesResults() {
+        navigateToReports();
+        EncorapediaReportsPage objEncorapediaReportsPage = new EncorapediaReportsPage(driver);
```

Comment:
This is more like a coding good practice. Declare the variables as near as possible of the place where you will use it.